plugins/v1/repository_s3/README.md

```diff
@@ -0,0 +1,28 @@
+This directory contains the (optional) keystore configuration for the `repository-s3` plugin.
+For more details on secure settings for the repository-s3 plugin please refer to the [repository-s3-client](https://www.elastic.co/guide/en/elasticsearch/plugins/current/repository-s3-client.html) documentation.
+
+### Parameters
```

Comment:
The documentation for the mandatory parameter `client_name` is missing. IMHO we should also rename that parameter to `s3_client_name`.

Reply:
Fixed in db748c6098d76529876ef5deac3eb35c73c9a4a0